### PR TITLE
Remove prometheus metrics from cgroups

### PR DESCRIPTION
**Pull request by crosbymichael**
Removing the prom code from cgroups allows more flexibility for consumers to structure metrics how they need.

### CODE REVIEW

**Comment by stevvooe** [comment]
@crosbymichael Presumably, we are moving up the stack to assemble these metrics?

**Comment by crosbymichael** [comment]
@stevvooe ya, moving it into containerd

File: memory.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (m *memoryController) Update(path string, resources *specs.LinuxResources)
 	if resources.Memory == nil {
 		return nil
 	}
-	g := func(v *uint64) bool {
+	g := func(v *int64) bool {
```
Comment (justincormack) [inline memory.go]: These are already merged - rebase issue?

Comment (crosbymichael) [inline memory.go]: No they are not, not in this package

Comment (justincormack) [inline memory.go]: Ah yes, sorry!

**Pull request merged**